fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/executable/NumericArithmetic.java

```diff
@@ -596,4 +650,515 @@ public static Expression divideDecimalV3(DecimalV3Literal first, DecimalV3Litera
         return new DecimalV3Literal(DecimalV3Type.createDecimalV3TypeLooseCheck(
                 t1.getPrecision(), t1.getScale() - t2.getScale()), result);
     }
+
+    /**
+     * coalesce
+     */
+    @ExecFunction(name = "coalesce")
```

Comment:
why "coalesce" is numericArithmetic?

Reply:
actually it is not only numeric arithmetic, but it is hard to well classify, so I put it here